### ISSUE TITLE
VRSTests gets spammed with missing VK_VERSION_1_2 extension

### ISSUE THREAD

**Issue by GamesTrap** [GamesTrap/TRAP#107]
Running the VRSTests ~~on Linux~~ the terminal gets spammed with the following Vulkan error:

```
[22:23:03][Error][Renderer][Vulkan][Debug][UNASSIGNED-GeneralParameterError-ExtensionNotEnabled] Validation Error: [ UNASSIGNED-GeneralParameterError-ExtensionNotEnabled ] Object 0: handle = 0x555556c64d60, type = VK_OBJECT_TYPE_INSTANCE; | MessageID = 0xe92b452d | Attempted to call vkCmdSetFragmentShadingRateKHR() but its required extension VK_VERSION_1_2 has not been enabled
 (-383040211)
```

All necessary device and instance extensions are loaded correctly:
```
[22:24:32][Debug][Renderer][Vulkan][Instance] Loading Instance Layer(s):
[22:24:32][Debug][Renderer][Vulkan][Instance]     VK_LAYER_KHRONOS_validation
[22:24:32][Debug][Renderer][Vulkan][Instance] Loading Instance Extension(s):
[22:24:32][Debug][Renderer][Vulkan][Instance]     VK_KHR_surface
[22:24:32][Debug][Renderer][Vulkan][Instance]     VK_KHR_xcb_surface
[22:24:32][Debug][Renderer][Vulkan][Instance]     VK_EXT_debug_utils
[22:24:32][Debug][Renderer][Vulkan][Device] Loading Device Extension(s):
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_swapchain
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_draw_indirect_count
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_EXT_descriptor_indexing
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_EXT_fragment_shader_interlock
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_buffer_device_address
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_maintenance4
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_EXT_memory_budget
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_create_renderpass2
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_fragment_shading_rate
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_timeline_semaphore
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_multiview
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_create_renderpass2
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_shader_float_controls
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_deferred_host_operations
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_acceleration_structure
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_spirv_1_4
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_ray_tracing_pipeline
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_KHR_ray_query
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_NV_device_diagnostics_config
[22:24:32][Debug][Renderer][Vulkan][Device]     VK_NV_device_diagnostic_checkpoints
```

Although the terminal gets spammed the VRS implementation itself still works normally like before.

**Comment by GamesTrap** [comment]
~~Still happening with NVIDIA driver 530.41.03~~ _Vulkan bug (not NVIDIA specific)_

**Comment by GamesTrap** [comment]
Upstream issue:

- https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/5733

This bug should be fixed in the next Vulkan SDK release.

**Comment by GamesTrap** [comment]
TBD: This maybe fixed now with SDK 1.3.250.0.

Also set Build-CI back to latest Vulkan SDK version

**Comment by GamesTrap** [comment]
Fixed when using:
```
[22:17:12][Info][Renderer][Vulkan] ----------------------------------
[22:17:12][Info][Renderer][Vulkan] Vulkan:
[22:17:12][Info][Renderer][Vulkan] Instance Version: 1.3.250
[22:17:12][Info][Renderer][Vulkan] Driver Vulkan Version: 1.3.242
[22:17:12][Info][Renderer][Vulkan] Renderer: NVIDIA GeForce RTX 2080
[22:17:12][Info][Renderer][Vulkan] Driver: 535.216.192
[22:17:12][Info][Renderer][Vulkan] ----------------------------------
```

Build-CI reverted back to using latest Vulkan SDK
